.github/workflows/ci.yml

```
@@ -0,0 +1,49 @@
+name: babel-loader
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  lint:
+    name: lint
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node-version: [12.x]
```

```suggestion
        node-version: "*"
```
Nit: To align with the babel repo, use latest node for linting (so we can always bump linter versions).

---

Also: we don't need matrix for lint jobs, you can use
```
runs-on: ubuntu-latest
```

This job is also lack of a `setup-node` action.